### PR TITLE
Fixing build action triggers

### PR DESCRIPTION
**Pull request by erikbosch**
Rationale:
Kuksa-val build was triggered too often due to negative requirement and a spelling error
Clean-images is useless as we do not upload images tagged by PR

But then, should we not trigger when certs and proto files are changed as well (Even if some use symbolic links, do not know if build action is smart enough to follow symbolic links and check if they have been changed))

_Note: Not intended for v0.4!_

### CODE REVIEW

File: .github/workflows/kuksa_val_docker.yml [hunk 1]
```diff
@@ -5,15 +5,13 @@ on:
...
-    - "kuksa_databroker/**"
-    - "kuksa_go_client/**"
-    - "kuksa_client/**"
-    - "Cargo.*"
-    - "Cross.toml"
+    paths:
+    - ".github/workflows/kuksa_val_docker.yml"
+    - "kuksa-val-server/**"
+    - "kuksa_certificates/**"
+    - "proto/**"
```
Comment (lukasmittag) [inline .github/workflows/kuksa_val_docker.yml:12]: do we need proto here? Server doesn't use it or am I wrong?

**Review by lukasmittag — CHANGES_REQUESTED** [review]

File: .github/workflows/kuksa_val_docker.yml (hunk 1 above)
Comment (erikbosch) [reply .github/workflows/kuksa_val_docker.yml:12]: Correct, good catch, it uses it's own proto file at https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/protos/kuksa.proto. Will update

**Comment by erikbosch** [comment]
> I think its my bad the comment above. we have a grpc side for server too. Forgot this

But that is not in the Proto folder, it is in https://github.com/eclipse/kuksa.val/blob/master/kuksa-val-server/protos/kuksa.proto

**Comment by erikbosch** [comment]
The databroker failed build is related to #631

**Review by lukasmittag — APPROVED** [review]

**Comment by SebastianSchildt** [comment]
Pls rebase to master  to see it runs through now

**Pull request merged**